app/styles/ember-appuniversum/_s-root.scss

```diff
@@ -125,6 +125,11 @@
   --au-h1-medium: 4rem;
   --au-h1: 4.4rem;
 
+  // Icons
```

Comment:
@Dietr Is it ok to refactor this as CSS variables? Not entirely sure about this one as SCSS variables should also work with swapping the import order of components a bit.

 You can check `_c-icon.scss` and `_c-button.scss` for the implementation.